DeepMRSeg/deepmrseg_test.py

```diff
@@ -38,63 +43,57 @@ def read_flags():
 	# Only parse for "--config"
 	configarg,_ = parser.parse_known_args()
 
-#	INPUT LIST
+#	I/O
 #	==========
-	inputArgs = parser.add_argument_group( 'REQUIRED INPUT ARGS' )
+	inputArgs = parser.add_argument_group( 'INPUT SCANS - REQUIRED', 'Select one of the 3 options: Single case (inImg/outImg), process all scans in a folder (inDir/outDir/outSuff) or using a scan list (sList)')
+	
+	## I/O Option1: Single scan I/O
+	inputArgs.add_argument( "--inImg", action='append', default=None, type=str, \
+				help="I/O Option1: Input scan name(s)")
+	inputArgs.add_argument( "--outImg", default=None, type=str, \
+				help="I/O Option1: Output scan name")
+
+	## I/O Option2: Batch I/O from folder
+	inputArgs.add_argument( "--inDir", default=None, type=str, \
+				help="I/O Option2: Input folder name")
+	inputArgs.add_argument( "--outDir", default=None, type=str, \
+				help="I/O Option2: Output folder name")
+	inputArgs.add_argument( "--outSuff", default='_SEG', type=str, \
+				help="I/O Option2: Output suffix")
+
+	## I/O Option3: Batch I/O from list
 	inputArgs.add_argument( "--sList", default=None, type=str, \
-				help="absolute path to the subject list for training")
-	inputArgs.add_argument( "--idCol", default=None, type=str, \
-				help="id column name (e.g.: 'ID')" )
-	inputArgs.add_argument( "--refMod", default=None, type=str, \
-				help="referene modality column name (e.g.: 'FL')")
-	inputArgs.add_argument( "--otherMods", default=None, type=str, \
-				help="other modalities to use (e.g.: 'T1,T2')" )
-	inputArgs.add_argument( "--outCol", default=None, type=str, \
-				help="column name containing the output filepaths (e.g.: 'out')")
-	inputArgs.add_argument( "--roi", default=None, \
-				help="absolute path to the ROI csv containing the ROIs to be \
-					considered and their respective indices. If not provided, \
-					the ROIs are assumed to be in the range( 0,num_classes-1 )")
-
-#	DIRECTORIES
+				help="I/O Option3: Scan list")
+
+#	MODELS
 #	===========
-	dirArgs = parser.add_argument_group( 'DIRECTORIES' )
-	dirArgs.add_argument( "--mdlDir", nargs='+', default="model", type=str, \
```

Comment:
Gives the following error:

FileNotFoundError: [Errno 2] No such file or directory: '/cbica/comp_space/doshijim/tmp_deepmrseg_ge/test/configs/test_config.json'

Need to edit deepmrseg_train to write files in the configs directory. I have made the edits in my branch.

Reply:
This can be handled in another PR from my branch